docs/user-guide/filtering-layers.md

```diff
@@ -89,7 +97,16 @@ In order to filter a numerical filed matching the records *greater than* or *equ
 
 <img src="../img/filtering-layers/attribute-table-quick-filter-3.jpg" class="ms-docimage"/>
 
-The *Quick Filter* remains active as long as the [Attribute Table](attributes-table.md) is open but, unlike the *Advanced Search*, closing the [Attribute Table](attributes-table.md) it will not reappear anymore if the [Attribute Table](attributes-table.md) is re-opened in a second time.
+#### Quick Filter by geometries
+
+If the user wants to filter by geometries, he must activate the **Filter clicking on the map** <img src="../img/button/filter_geometry_button.jpg" class="ms-docbutton"/> button (the button became blue) and select the geometry he wants to filter. The list of records in the [Attribute Table](attributes-table.md) will be automatically filtered by matching with the selected geometry. 
```

Comment:
If the user wants to filter records in the Attribute Table simply by clicking on the map, he can activate the **Filter clicking on the map** <img src="../img/button/filter_geometry_button.jpg" class="ms-docbutton"/> button (the button became blue) and then click on the map over the features he need to select. The list of records in the [Attribute Table](attributes-table.md) will be automatically filtered by matching with the selected point in the map.